config/routes.rb

```diff
@@ -5,7 +5,15 @@
   # root "articles#index"
    root "users#index"
 
+  get 'posts/:post_id/comments/new', to: 'comments#new'
+post 'posts/:post_id/comments', to: 'comments#create'
+post 'posts/:post_id', to: 'likes#create'
+  post 'posts', to: 'posts#create'
+  
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
+     resources :comments, only: [:new, :create, :destroy]
+      resources :likes, only: [:create]
+    end
```

Comment:
- [ ] This approach of defining routes and the ones above all work the same, I will suggest you make use of one and remove the other from your codebase to avoid redundancy :wink: